converters/cfloop.cfc

```diff
@@ -47,6 +47,18 @@ component extends="BaseConverter" {
 				s = s & "++";
 			}
 			s = s & " ) {";
+		} else if (structKeyExists(attr, "query")) {
+			s = "cfloop( query= " & attr.query;
+			if (structKeyExists(attr, "startrow")) {
+				s = s & " startRow= " & unPound(attr.startrow);
```

Comment:
ACF requires commas to separate script attributes, but I would recommend using something generic and already implemented as I suggested above.